pkg/project/project.go

```diff
@@ -366,6 +366,13 @@ func (p *Project) loadFromDirectory(
 
 	err := util.Walk(p.fs, ".", func(path string, info fs.FileInfo, err error) error {
 		if err != nil {
+			if errors.Is(err, os.ErrPermission) {
```

Comment:
you should be able to unnest this if since it early returns and a `nil` value for error should result in a false, no?

Reply:
also, if we have more than a single condition to check the `switch err.(type)` pattern is likely more idiomatic in Golang